spec/rubocop/cop/naming/predicate_name_spec.rb

```diff
@@ -73,4 +73,57 @@ def is_a?
       RUBY
     end
   end
+
+  context 'with method define macros' do
+    let(:cop_config) do
+      { 'NamePrefix' => %w[is_], 'NamePrefixBlacklist' => %w[is_],
+        'MethodDefineMacros' => %w[define_method def_node_matcher] }
+    end
+
+    it 'registers an offense when using `define_method`' do
+      expect_offense(<<-RUBY.strip_indent)
+        define_method(:is_hello) do |method_name|
+                      ^^^^^^^^^ Rename `is_hello` to `hello?`.
+          method_name == 'hello'
+        end
+      RUBY
+    end
+
+    it 'registers an offense when using an internal affair macro' do
+      expect_offense(<<-RUBY.strip_indent)
+        def_node_matcher :is_hello, <<-PATTERN
+                         ^^^^^^^^^ Rename `is_hello` to `hello?`.
+          (send
+             (send nil? :method_name) :==
+             (str 'hello'))
+        PATTERN
+      RUBY
+    end
+  end
+
+  context 'without method define macros' do
```

Comment:
same here

Reply:
Btw, it seems you copy/pasted the previous context here (or I'm missing something).

Reply:
It is written as `with` previous context, 'without' here context.